style.css

```diff
@@ -338,7 +338,213 @@ textarea {
   gap: 50px;
 }
 
+/* Style of popup added through javascript Mobile */
+.work-info {
+  display: grid;
+  padding: 16px;
+  font-size: 16px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transition: 2000ms ease-in-out;
+  transform: translate(-50%, -50%) scale(1);
+  height: 100%;
+  width: 100%;
+  z-index: 2;
+}
+
+.cancel-btn {
+  position: absolute;
+  top: 4.1%;
+  right: 7%;
+  background: none;
+  border: none;
+}
+
+.cancel-btn img {
+  width: 35px;
+  height: 35px;
+  visibility: hidden;
+}
+
+.work-image {
+  overflow: hidden;
+  box-sizing: border-box;
+  border-radius: 16px;
+}
+
+.work-image > img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+.work-title {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 2.6em;
+  color: #172b4d;
+  text-align: left;
+  margin-top: 0.5em;
+}
+
+.tech-list {
+  list-style: none;
+  padding: 0;
+  display: flex;
+  justify-content: flex-start;
+  gap: 1em;
+  margin: 1em 0;
+}
+
+.list1 {
+  background: #ebf0ee;
+  color: #3a4a42;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 0.95em;
+  line-height: 1em;
+  letter-spacing: 0.03em;
+  border-radius: 0.25em;
+  padding: 0.8em;
+}
+
+.description {
+  color: #344563;
+  text-align: left;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 1.2em;
+  line-height: 1.5em;
+}
+
+.work-buttons {
+  display: flex;
+  justify-content: space-between;
+}
+
+.wrk-btn {
+  display: flex;
+  margin-top: 1.5em;
+  margin-bottom: 1.5em;
+  border: none;
+  align-self: center;
+  background: #36b37f;
+  font-weight: 500;
+  font-size: 16px;
+  letter-spacing: 2px;
+  padding: 1em 1.5em;
+  color: white;
+  border-radius: 3px;
+  width: 45%;
+}
+
+.wrk-btn > img {
+  margin-left: 1.3em;
+}
+
+.Desktop-img {
+  display: none;
+}
+
 @media (min-width: 768px) {
+  /* Pop design added through javaScript css */
+  .Mobile-img {
+    display: none;
+  }
+
+  .Desktop-img {
+    display: initial;
+  }
+
+  .work-info {
+    display: grid;
+    height: 90%;
+    width: 30%;
+    grid-template-columns: 1fr 1fr 1.2fr 60px;
+    grid-template-rows: 30px 50% 2fr 20px 2fr;
+  }
```

Comment:
In your about me section your content seem to be overlappin see image below to see what i mean.

This is your design: 

![Screen 5](https://user-images.githubusercontent.com/79506105/188899652-a2c89b3b-8c28-46df-b97a-3c71761af4e5.png)

Stick to this design: 
![Screen Shot 6](https://user-images.githubusercontent.com/79506105/188899802-9311753d-8278-48c9-b20a-826343e91dbf.png)
 
 You can fix ths issue by add the fooling styling your about-me-card class in your css like this
Example:

.about-me-card {
    grid-template: 1fr/1fr 1fr 1fr;
    gap: 3%;
    margin-top: 10vh;
    padding: 0 20%;
}

So in this case you need to adjust the margin-top to 10vh

Reply:
Thank you. adjusted